helm/sealed-secrets/templates/deployment.yaml

```
@@ -57,6 +57,12 @@ spec:
           volumeMounts:
           - mountPath: /tmp
             name: tmp
+          {{- if .Values.env }}
+          env:
+            {{- with .Values.env }}
+              {{- toYaml . | nindent 12 }}
+            {{- end }}
+          {{- end }}
```

This is my suggestion for supporting custom env vars from different sources:
```suggestion
         {{- if .Values.extraEnvVars }}
         env: {{- include "sealed-secrets.render" (dict "value" .Values.extraEnvVars "context" $) | nindent 12 }}
          {{- end }}
          {{- if or .Values.extraEnvVarsCM .Values.extraEnvVarsSecret }}
          envFrom:
            {{- if .Values.extraEnvVarsCM }}
            - configMapRef:
                name: {{ .Values.extraEnvVarsCM }}
            {{- end }}
            {{- if .Values.extraEnvVarsSecret }}
            - secretRef:
                name: {{ .Values.extraEnvVarsSecret }}
            {{- end }}
          {{- end }}
```